keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -101,12 +102,20 @@ this enhancement is being considered for a milestone.
 
 ## Summary
 
-When a cluster has multiple apiservers at mixed versions (such as during an
-upgrade/downgrade or when runtime-config changes and a rollout happens), not
-every apiserver can serve every resource at every version.
+This proposal introduces a Mixed Version Proxy (also earlier referred to as Unknown Version Interoperability Proxy) to solve issues with version skew in Kubernetes clusters. During upgrades or downgrades, when API servers have different versions, this feature ensures that:
 
-To fix this, we will add a filter to the handler chain in the aggregator which
-proxies clients to an apiserver that is capable of handling their request.
+1. Client requests for a specific built-in resource are proxied to an API server capable of serving it, avoiding 404 Not Found errors
+2. Clients receive a complete, cluster-wide discovery document by merging information from all peer API servers, preventing controllers from making incorrect decisions based on incomplete data
+
+Merged discovery is only supported for aggregated discovery endpoint, which requires clients to use the aggregated discovery Accept headers. Requests for un-aggregated (legacy) discovery will always return local-only data and do not participate in peer merging. All discovery changes are implemented at the existing aggregated discovery endpoint `/apis`, with no new discovery endpoints being introduced.
+
+**Note:** Merged discovery is not supported for the api endpoint (`/api`) that serves the `core/v1` group. Since Kubernetes v1.4, no new top-level types have been added to `core/v1` group; only subresources have been introduced:
```

Comment:
@Jefftree @liggitt @jpbetz for this Note. 
From previous chat about this, we decided that we will not support merged discovery for `/api`, just confirming that there's still agreement on that.